test/config/cassdc/kustomization.yaml

```diff
@@ -1,5 +1,8 @@
+namespace: deploy-reaper-test
+
 resources:
 - cassdc.yaml
+- cassdc-non-reaper.yaml
```

Comment:
See my comment below about creating a separate test. Assuming we do that we will want to create some new kustomize templates. We might create something like this under `test/config/cassdc`:

```
├── kustomization.yaml
├── bases
├── test-a
├── test-b
└── test-c
```
The `bases` directory would have the common CassandraDatacenter template. Each `test-` directory would extend the base. 

Then under the `test/config` directory we might have:

```
├── cass-operator
├── cassdc
└── tests
    ├── test-a
    └── test-b
```
where each `test-` subdirectory would have its own `kustomization.yaml` file. I need to think about this some more.